Mathlib/Order/Hom/CompleteLattice.lean

```diff
@@ -193,9 +193,6 @@ instance (priority := 100) OrderIsoClass.tosInfHomClass [CompleteLattice α]
 -- See note [lower instance priority]
 instance (priority := 100) OrderIsoClass.toCompleteLatticeHomClass [CompleteLattice α]
     [CompleteLattice β] [OrderIsoClass F α β] : CompleteLatticeHomClass F α β :=
-  -- Porting note: Used to be:
-    -- { OrderIsoClass.tosSupHomClass, OrderIsoClass.toLatticeHomClass,
-    -- show sInfHomClass F α β from inferInstance with }
   { OrderIsoClass.tosSupHomClass, OrderIsoClass.tosInfHomClass with }
```

Comment:
I assume you checked that the missing `show sInfHomClass F α β from inferInstance` isn't causing any defeq issues?

Reply:
Which diamond are you concerned about?

Reply:
The definition would copy fields differently. But I guess this would be irrelevant either way since the only data-carrying field is `toFun` here and that would be set by `OrderIsoClass.tosSupHomClass`.